guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryView.java

```diff
@@ -0,0 +1,87 @@
+package org.guvnor.structure.client.editors.repository.clone;
+
+import com.github.gwtbootstrap.client.ui.constants.ControlGroupType;
+
+public interface CloneRepositoryView {
```

Comment:
Not applicable because of CoreConstants used in view. HasBusyIndicator interface has method for showing busy state, which takes message parameter. This parameter is useless because I use CoreConstants in view.